be/src/storage/lake/tablet_manager.cpp

```diff
@@ -2,58 +2,143 @@
 
 #include "storage/lake/tablet_manager.h"
 
+#include "fmt/format.h"
+#include "fs/fs.h"
+#include "gen_cpp/AgentService_types.h"
+#include "gen_cpp/starlake.pb.h"
+#include "gutil/strings/util.h"
+#include "re2/re2.h"
+#include "service/staros_worker.h"
 #include "storage/lake/group_assigner.h"
 #include "storage/lake/metadata_iterator.h"
 #include "storage/lake/tablet.h"
 #include "storage/lake/tablet_metadata.h"
 #include "storage/lake/txn_log.h"
 #include "util/lru_cache.h"
+#include "util/raw_container.h"
+#include "util/string_parser.hpp"
 
 namespace starrocks::lake {
 
 TabletManager::TabletManager(GroupAssigner* group_assigner, int64_t cache_capacity)
         : _group_assigner(group_assigner), _metacache(new_lru_cache(cache_capacity)) {}
 
+// path $bucket:/$ServiceID/$GroupID/tbl_${TabletID}_${version}
+std::string TabletManager::tablet_meta_path(const std::string& group, int64_t tablet_id, int64_t verson) {
+    return fmt::format("{}/tbl_{:016X}_{:016X}", group, tablet_id, verson);
+}
+
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+std::string TabletManager::txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("{}/txn_{:016X}_{:016X}", group, tablet_id, txn_id);
+}
+
 Status TabletManager::create_tablet(const TCreateTabletReq& req) {
-    (void)req;
-    return Status::NotSupported("TabletManager::create_tablet");
+    // generate metapb
+    TabletMetadataPB tablet_metadata_pb;
+    tablet_metadata_pb.set_id(req.tablet_id);
+    tablet_metadata_pb.set_version(1);
+
+    // rowset metadata with version 1
+    auto rowset_metadata_pb = tablet_metadata_pb.add_rowsets();
+    rowset_metadata_pb->set_id(1);
+    rowset_metadata_pb->set_overlapped(false);
+    rowset_metadata_pb->set_data_size(0);
+    rowset_metadata_pb->set_num_rows(0);
+    tablet_metadata_pb.set_next_rowset_id(2);
+
+    // get shard group
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(req.tablet_id));
+
+    // write tablet meta
+    auto meta_path = tablet_meta_path(group_path, tablet_metadata_pb.id(), tablet_metadata_pb.version());
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(meta_path));
+    ASSIGN_OR_RETURN(auto wf, fs->new_writable_file(meta_path));
+    RETURN_IF_ERROR(wf->append(tablet_metadata_pb.SerializeAsString()));
+    RETURN_IF_ERROR(wf->close());
+    return Status::OK();
 }
 
 StatusOr<Tablet> TabletManager::get_tablet(int64_t tablet_id) {
-    (void)tablet_id;
-    return Status::NotSupported("TabletManager::get_tablet");
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(tablet_id));
```

Comment:
```suggestion
    ASSIGN_OR_RETURN(auto group_path, _group_assigner->get_group(tablet_id));
```
Replace all `get_staros_shard_path` with `_group_assigner->get_group`.